src/transform/schema.ts

```diff
@@ -45,6 +45,19 @@ export function transformSchemaObjMap(obj: Record<string, any>, options: Transfo
   return output.replace(/\n+$/, "\n"); // replace repeat line endings with only one
 }
 
+/** make sure all required fields exist **/
+export function addRequiredProps(properties: Record<string, any>, required: string[]): string[] {
+  const missingRequired = required.filter((r: string) => !(r in properties));
+  if (missingRequired.length == 0) {
+    return [];
+  }
+  let output = "";
+  for (const r of missingRequired) {
+    output += `${r}: any;\n`;
```

Comment:
In anticipation of #625 (not merged yet), let’s use `unknown` instead of `any` here

Reply:
```suggestion
    output += `${r}: unknown;\n`;
```